lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -1680,8 +1716,18 @@ def loki_endpoints(self) -> List[dict]:
             A list with Loki Push API endpoints.
```

Comment:
List of strings or list of dicts? I.e. does the return type annotation needs an update?
(And why did mypy not complain..?)

Reply:
List of dicts:

```python
[{"url": "http://endpoint1"}, "url": "http://endpoint2"}]
```